src/FSharp.Analyzers.SDK/FSharp.Analyzers.SDK.fsproj

```diff
@@ -1,25 +1,26 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
-  </PropertyGroup>
-  <PropertyGroup>
-    <PackageId>FSharp.Analyzers.SDK</PackageId>
-    <IsPackable>true</IsPackable>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <Version>0.11.0</Version>
     <Authors>Krzysztof Cieslak</Authors>
-    <PackageDescription>SDK for building custom analyzers for FSAC / F# editors</PackageDescription>
+    <Description>SDK for building custom analyzers for FSAC / F# editors</Description>
     <Copyright>Copyright 2019 Lambda Factory</Copyright>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageTags>F#;analyzers;compiler;tooling;editor;</PackageTags>
-    <RepositoryUrl>https://github.com/ionide/FSharp.Analyzers.SDK</RepositoryUrl>
-    <RepositoryType>git</RepositoryType>
-    <RepositoryBranch>master</RepositoryBranch>
-    <DebugType>embedded</DebugType>
+    <PackageReleaseNotes>https://github.com/ionide/FSharp.Analyzers.SDK/blob/master/RELEASE_NOTES.md</PackageReleaseNotes>
```

Comment:
Same comment about release notes here - the target could even be shared and placed in `Directory.Build.targets`.